package.json

```
@@ -16,7 +16,9 @@
     "eslint-config-htmlacademy": "2.0.0"
   },
   "license": "MIT",
-  "dependencies": {},
+  "dependencies": {
+    "lodash": "4.17.20"
```

```suggestion
    "lodash": "4.17.20"
```
Зачем лодаш?)

Как минимум он не используется 🙂